src/transaction/payment.ts

```diff
@@ -85,29 +85,57 @@ function createMaximalAmount(amount: Amount): Amount {
   return _.assign({}, amount, {value: maxValue})
 }
 
+function normalizeAddress(address: string, tag: number | undefined, verification?: string):
```

Comment:
Please document this function - it has a fine name but the functionality is non-obvious